### DISCUSSION TITLE
PDF Generator

### DISCUSSION THREAD

**Discussion by eflashlogistics** [code4recovery/12-step-meeting-list#1434] [Q&A]
Hi. Two things

1. Generated pdf is showing back end values for meeting types: Example XYZ
2. Will it ever be possible to have them generate a meeting list from the front end? I always have to come back and train someone how to generate to a pdf, download, upload to media, change our link on the front end. 

Thanks!

**Comment by ginslo** [comment]
Hello @eflashlogistics . To help us fully understand the issue, can you post the PDF or a screenshot which shows the issue please? 

For the 2nd point, I am not aware of any plans for a front end generator. I understand the motivation, but my guess is that there are too many variations with the way different organizations put together their publications in terms of front and end pages etc. that it would not be possible or practical to implement.
